ios/keyman/Keyman/Keyman/MainViewController.swift

```diff
@@ -130,7 +128,6 @@ class MainViewController: UIViewController, TextViewDelegate, UIActionSheetDeleg
     super.viewDidLoad()
 
     extendedLayoutIncludesOpaqueBars = true
-    automaticallyAdjustsScrollViewInsets = false
```

Comment:
<img width="782" alt="Screen Shot 2021-05-27 at 2 34 18 PM" src="https://user-images.githubusercontent.com/25213402/119785350-52c97280-bef9-11eb-8f0a-dc215de0af2e.png">

I was unable to find any UIScrollVIew or UIContainerView instances that would have been the target of this statement.  It may be an artifact from long-removed code...?  At any rate, I didn't notice any difference in behavior after removing this line.